content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -252,6 +252,10 @@ If you are using a different CRI, you have to modify the file
 ```bash
 KUBELET_EXTRA_ARGS=--cgroup-driver=<value>
 ```
+{{< warning >}}
+Flag --cgroup-driver has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
```

Comment:
If `--cgroup-driver` is a deprecated flag, then the documentation shouldn't recommend setting it (on line 253).

This doesn't look like the right fix.

Reply:
i'd say omit this change for now.

while some of the kubelet flags are deprecated there is no clear timeline when they are going to be removed. so, once the time comes we will update both their handling in kubeadm and the documentation.

Reply:
sure

Reply:
@RA489 any update on this?